opm/simulators/flow/SimulatorFullyImplicitBlackoil.hpp

```diff
@@ -468,7 +468,7 @@ class SimulatorFullyImplicitBlackoil : private SerializableSim
             simulator_.problem().setSimulationReport(report_);
         } else {
             // solve for complete report step
-            auto stepReport = solver_->step(timer);
+            auto stepReport = solver_->step(timer, adaptiveTimeStepping_->timeStepControl());
```

Comment:
I believe this line is the problem. It assumes there always is a non-null adaptiveTimeStepping_ member, which is false: if you run with `--enable-adaptive-time-stepping=false` it is indeed a null pointer. So a slightly different approach is needed.

Reply:
>  So a slightly different approach is needed.

We should obviously always check for null, but there's likely to be a deeper problem here too.  If it were "only" `--enable-adaptive-time-stepping=false` that got us into this situation, then the problem would not consistently show up for restarted runs, and mostly in parallel.

Reply:
the crash set is exactly those tests that use --enable-adaptive-time-stepping=false.